taskcluster/scripts/requirements.in

```diff
@@ -0,0 +1,5 @@
+# These requirements are used in build scripts
+
+multidict>=5.0.0 
+yarl>=1.6.0
+taskcluster==53.1.0
```

Comment:
Note I'm upgrading `taskcluster` from `52.0.0` to `53.1.0`
I'm not sure if pinning and upgrading packages belong in the same PR. Let me know if we should split them.